hedera-node/test-clients/record-snapshots/CryptoTransfer-SpecialAccountsBalanceCheck.json

```diff
@@ -0,0 +1 @@
+{"placeholderNum":1008,"encodedItems":[]}
```

Comment:
No items here. Do we still need this to be generated or this is an example to handle empty item?

Reply:
Hi @iwsimon , this spec [doesn't actually submit any transactions](https://github.com/hashgraph/hedera-services/blob/09697-auto-snapshot-matching/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java#L1816), haha.